src/lib/stitching/convection/link.ts

```diff
@@ -0,0 +1,38 @@
+import urljoin from "url-join"
+
+import { ApolloLink } from "apollo-link"
+import { setContext } from "apollo-link-context"
+import { createHttpLink } from "apollo-link-http"
+import config from "config"
+import { headers as requestIDHeaders } from "lib/requestIDs"
+import fetch from "node-fetch"
+
+const { CONVECTION_API_BASE } = config
+
+export const createConvectionLink = () => {
+  const httpLink = createHttpLink({
+    fetch,
+    uri: urljoin(CONVECTION_API_BASE, "graphql"),
+  })
+
+  const middlewareLink = new ApolloLink(
+    (operation, forward) => (forward && operation && forward(operation)) || null
```

Comment:
I think I remember dealing with this at some point, but the code is so vague… can we hide that in a shared function that has some explanation of what this does?